CODE/Sources/PennyBOT/Constants.swift

```diff
@@ -35,5 +35,12 @@ enum Constants {
             .moderator,
             .core,
         ]
+
+        static let elevatedRestrictedCommandsAccess: [Roles] = [
```

Comment:
Help command add/remove is only accessible to these roles.
We can also put a setting using Discord clients so these are enforced by Discord too.